pykickstart/commands/clearpart.py

```diff
@@ -151,7 +151,23 @@ def _getParser(self):
                         and gpt for x86_64 but not dasd.""")
         return op
 
-class F28_ClearPart(F21_ClearPart):
+class F25_ClearPart(F21_ClearPart):
+    def _getParser(self):
+        op = super(F25_ClearPart, self)._getParser()
+        op.add_argument("--drives", type=commaSplit, version=F25, help="""
+                        You can combine multiple ways of specifying storage
+                        targets in one command, using the pipe ("|") as a
+                        delimiter. For example:
+
+                        ``clearpart --drives=sd*|hd*|vda,/dev/vdc``
+
+                        The above will match either of sd*, hd*, or vda and
+                        /dev/vdc. Variants are processed from left to right,
+                        and at least one of them must match for the command
+                        to succeed.""")
```

Comment:
I must say, it is not at all clear what is the difference between `|` and `,`. It sounds like one is "or", while the other is "and", but the text says "vda and /dev/vdc", and no disk will be simultaneously named "vda" and "/dev/vdc", so it seems that the `vda,/dev/vdc` part will be simply ignored (will never match).

Reply:
I had the same feeling so the message is definitely not great. We will try to improve the documentation the make the difference obvious.

In short, the difference is that `|` will match X-N disks for the whole expression. However, `,` must match all the time. In other words:

`sd*|vd*` could match - `vda`
`sda,vda`  have to match `sda` but also `vda`

Reply:
The option takes a list of drives. You specify that as a list of patterns, separated by comma `,` . Each pattern can consist of multiple alternatives, separated by pipe `|`. Patterns can also use asterisk `*` for wildcard matching.

As an example, `ignoredisk --drives=sd*|hd*|vda,/dev/vdc` has one comma, so it lists two drives: The first is either `sd*`, or `hd*`, or `vda`. The second is `/dev/vdc`.